src/client/Microsoft.Identity.Client/ManagedIdentity/V2/Cache/Abstractions/IBindingCache.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Identity.Client.ManagedIdentity.V2.Cache.Abstractions
+{
+    internal interface IBindingCache
+    {
+        void Cache(string identityKey, string tokenType,
+                    CertificateRequestResponse resp, string subject);
+
+        bool TryGet(string identityKey, string tokenType,
+                    out CertificateRequestResponse resp, out string subject);
```

Comment:
Do we really need to cache the entire CertificateRequestResponse? It seems very heavy.